.golangci.yml

```diff
@@ -1,9 +1,20 @@
 run:
   tests: false
+  skip-dirs:
+    - contrib
+    - compose
+    - graphql/bench
+    - graphql/e2e
+    - graphql/test
+    - graphql/testdata
+    - systest
+    - t
+  skip-files:
+    - ".*test.go$"
```

Comment:
I think this should be ".*_test.go$"

From [https://go.dev/doc/tutorial/add-a-test](https://go.dev/doc/tutorial/add-a-test): "Ending a file's name with _test.go tells the go test command that this file contains test functions."

Reply:
actually they have this flag set here for what you are referring to https://github.com/dgraph-io/dgraph/pull/8187/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R2, but the issue I noticed was some test-files don't follow that pattern for that flag to take effect. Hence, I introduced the `*test.go` vs. `*_test.go`.

Reply:
I'm resolving this, because the `regex` `*test.go` is a superset to `*_test.go`. I hope that is okay.